tests/backstop_data/gulp-tasks/setup-data.js

```diff
@@ -0,0 +1,33 @@
+const relationshipTypeService = require('./data-setup-steps/relationship-type.service');
+const caseTypeService = require('./data-setup-steps/case-type.service.js');
+const contactService = require('./data-setup-steps/contact.service.js');
+const activityService = require('./data-setup-steps/activity.service.js');
+const relationshipService = require('./data-setup-steps/relationship.service.js');
+const caseService = require('./data-setup-steps/case.service.js');
+const customGroupService = require('./data-setup-steps/custom-group.service.js');
+const customFieldService = require('./data-setup-steps/custom-field.service.js');
+const tagService = require('./data-setup-steps/tag.service.js');
+const createSampleUploadFile = require('./data-setup-steps/create-sample-upload-file.service.js');
+
+module.exports = setupData;
+
+/**
+ * Setups the data needed for some of the backstop tests.
+ *
+ * @returns {Promise} An empty promise that is resolved when the task is done.
+ */
+async function setupData () {
+  createSampleUploadFile();
+
+  relationshipTypeService.setupData();
+  caseTypeService.setupData();
+  contactService.setupData();
+  caseService.setupData();
+  activityService.setupData();
+  await relationshipService.setupData();
```

Comment:
Because we have the sleep function inside.

Reply:
@deb1990 since we are trying to mimic the same interface for all of these service we can't have a special one. `setupData` should always return a promise and we should wait for all of them. I know that we are waiting here on this one because some later setups depend on this one.

Let's not do anything about this right now, but we need to think of a way to explicitly explain that this particular service is a dependency of other services. Finding a random `await` can be confusing, specially in the future when we forget about this.